tests/widget/modal/widget-modal.login-with-misc-callbacks.test.js

```diff
@@ -21,15 +22,21 @@ test('Modal widget with simple login and misc callbacks', async ({ page }) => {
   await expect(page.getByText('Are you human?')).toBeVisible();
   await clickButton('Yes', '/authenticate'); // <- Self submitting callback
 
-  // Choice
+  // Choice (select)
   const selectEl = page.getByLabel('Are you sure?');
   await selectEl.selectOption('0');
 
-  // Polling Wait
+  // Choice (radio)
+  const radioEl = page.getByLabel('Are you sure?');
+  await radioEl.selectOption('0');
+  await clickButton('Next', '/authenticate');
+
   await Promise.all([
     // NOTE: Make sure timer is same or more than set in Polling Wait node
     page.waitForTimeout(3000),
     page.locator('button', { hasText: 'Next' }).click(),
+    // Spinner
+    expect(page.getByRole('status')).toBeVisible(),
```

Comment:
This was the best way I could think of to check that the spinner was visible, using the ARIA status role.

Reply:
This is good. I like it.